.pre-commit-config.yaml

```diff
@@ -0,0 +1,5 @@
+repos:
```

Comment:
It probably would be worth adding a quick note within the `README.md` or in a `CONTRIBUTING.md` file telling contributors how they can verify pre commits locally before opening a PR. To reduce the amount of errors potentially found when PR is opened.